res/furnace.rc

```diff
@@ -1 +1,41 @@
+1 VERSIONINFO
+    FILEVERSION 0,6,5,0
+    PRODUCTVERSION 0,6,5,0
+{
+    BLOCK "VarFileInfo"
+    {
+        VALUE "Translation",

```

Comment:
Language - Furnace supports more than English.

Reply:
I should maybe add more fileinfo resource blocks if that indeed is how to add multilanguage support to it

Reply:
turns out I just needed to add more values to the `Translation` key

the metadata files for other OSes so far is only English for now so I'm leaving the `StringFileInfo` block